shim/src/realm.js

```diff
@@ -72,6 +71,16 @@ function buildChildRealm(BaseRealm) {
     evaluate(...args) {
       return doAndWrapError(() => descs.evaluate.value.apply(this, args));
     }
+    static makeRootRealm() {
```

Comment:
Late to the party here, but I wonder why are we adding new APIs to the shim that are not part of the realm spec, instead of just adding them to an abstraction layer on top of the shim?